# DESCRIPTION: I have added comments in the given code to help user understand output type hint
@@ -1,2 +1,2 @@
-def print_hello_world() -> str:
-    return 'Hello World!'
+def print_hello_world() -> str:  # <------------- The "-> str" part defines the type of the returned variable as string
+    return 'Hello World!'  # <------------- Indeed, the output value is a string
